This patch disables the test set default verify paths, which tries to connect
to an external server. This tries to connect using s_client, which does
not work within OWAN, even if proxy vaeriables are set. 

--- pyOpenSSL-16.2.0/tests/test_ssl.py.~1~	2016-10-15 06:15:43.000000000 +0000
+++ pyOpenSSL-16.2.0/tests/test_ssl.py	2017-01-13 15:51:01.552602423 +0000
@@ -1183,6 +1183,10 @@
         reason="set_default_verify_paths appears not to work on Windows.  "
         "See LP#404343 and LP#404344."
     )
+    @pytest.mark.skipif(
+        True,
+        reason="This test doesn't work within OWAN, even with proxy set"
+    )
     def test_set_default_verify_paths(self):
         """
         :py:obj:`Context.set_default_verify_paths` causes the
